task8/script.js

```diff
@@ -0,0 +1,61 @@
+const documentFragment = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
```

Comment:
IMO
The constant we are declaring here is only used in line 26, right? To reduce unnecessary bugs, declare it where you will use it, instead of globally.

Reply:
I never considered where to declare the function.
Certainly, if you use it only in that place, I thought it would be better to make it inaccessible from others.

commit: 53fedfd